### PR TITLE
Enhancement

### PR DESCRIPTION
**Pull request by cs-sujeet-ahirrao**
- Add Validations in every page of wizard
- Handle multiple source control integrations
- As per the selection of selected source control show respective the connector configuration step

### CODE REVIEW

**Review by cs-pranali-chavan — APPROVED** [review]
Looks appropriate

File: widget/view.controller.js [hunk 1]
```diff
@@ -18,55 +16,59 @@
     $scope.close = close;
     $scope.moveNext = moveNext;
     $scope.movePrevious = movePrevious;
+    $scope.moveEnvPrevious = moveEnvPrevious;
     $scope.moveEnvironmentNext = moveEnvironmentNext;
     $scope.moveVersionControlNext = moveVersionControlNext;
-    $scope.versionControlConnectorName = 'github';
-    $scope.connectorVersion = '100.9.9';
+    $scope.moveNextWithoutSouruceContol = moveNextWithoutSouruceContol;
```
Comment (cs-suyog-jinturkar) [inline widget/view.controller.js]: moveNextWithoutSouruceContol --> moveNextWithoutSourceControl

File: widget/view.controller.js [hunk 2]
```diff
@@ -90,92 +93,140 @@
...
             $scope.removeDefaultFromOthers();
           }
 
         }
         $scope.formHolder.connectorForm.$setPristine();
-        if(!deleteConfig) {
+        if (!deleteConfig) {
           $scope.input.selectedConfiguration.id = response.id;
+          $scope.configuredConnector = true;
+          $scope.connectorHealth = true;
```
Comment (cs-suyog-jinturkar) [inline widget/view.controller.js]: connectorHealth --> isConnectorHealthy
configuredConnector --> isConnectorConfigured

File: widget/view.controller.js [hunk 3]
```diff
@@ -90,92 +93,140 @@
...
+          $scope.contentDetail = response.data;
+          connectorService.getConnectorHealth(response.data, connector.configuration[0].config_id, connector.configuration[0].agent).then(function (data) {
+            if (data.status === "Available") {
+              $scope.connectorHealth = true;
+            }
+          });
         });
+        if (!connector) {
+          toaster.error({
+            body: 'The Connector "' + connectorName + '" is not installed. Istall the connector and re-run thiz wizard to complete the configuration'
```
Comment (cs-suyog-jinturkar) [inline widget/view.controller.js]: Istall --> Install

Comment (cs-suyog-jinturkar) [reply widget/view.controller.js]: thiz ---> this

**Pull request merged**